package.json

```diff
@@ -26,7 +26,7 @@
     "react-apexcharts": "1.4.0",
     "react-content-loader": "6.2.0",
     "react-dom": "17.0.2",
-    "react-markdown": "8.0.3",
+    "react-markdown": "^8.0.3",
```

Comment:
Doesn't matter too much, but renovate will revert that one (same for the lock file)